src/work-allocation-2/models/cases/case-action.model.ts

```diff
@@ -0,0 +1,4 @@
+export default interface CaseAction {
```

Comment:
_Question:_
Is this identical to `TaskAction`? Is there scope for a more generic `WorkAction`?

Reply:
To avoid complaints about merge conflicts  I suggest looking at this at a future date as it would involve renaming in numerous task files.